nautobot/project-static/js/theme.js

```diff
@@ -83,6 +83,7 @@ function detectThemeSettings() {
 function setDarkTheme() {
     // Set theme to dark
     htmlEl.dataset.theme = "dark";
```

Comment:
Any idea if we still need the `theme` property for now? Guessing eventually just the `bsTheme` should be sufficient, but maybe that's future work?